node/src/chain/header_validation.rs

```diff
@@ -169,7 +174,7 @@ impl<'a, DB: database::DB> Validator<'a, DB> {
             Ok::<_, anyhow::Error>(prior_tip.header().seed)
         })?;
 
-        verify_block_att(
+        let (_, _, v_committee, r_committee) = verify_block_att(
```

Comment:
It is already generalized.  Both `verify_winning_att` and `verify_prev_block_cert` are already using same function `verify_block_att`.

Reply:
By generalize I mean they should be the same function.
An alternative option is to move `merge_committees` inside `verify_block_att`, since it's always called after it.